src/PowerShellEditorServices.Protocol/Server/LanguageServerSettings.cs

```diff
@@ -7,6 +7,8 @@ namespace Microsoft.PowerShell.EditorServices.Protocol.Server
 {
     public class LanguageServerSettings
     {
+        public bool LoadProfiles { get; set; }
```

Comment:
Is this the right setting name for what the user would specify in VS Code?  I used the positive form rather than the negative form (`noProfile`) since profile loading is never performed by default.